arrayHelper.js

```diff
@@ -0,0 +1,62 @@
+module.exports = {
+    /**
+     * Takes an array of strings, and transforms it into a single string for use in the database
+     * None of the strings should contain the preserved character combination #,#
+     * @param array
+     * @returns {string}
+     */
+    stringifyArrayOfStrings: function (array) {
+        if (array.length === 0) return "";
+
+        let result = "";
+        for (let i = 0; i < array.length - 1; i++) {
+            result += array[i];
+            result += "#,#";
+        }
+        result += array[array.length - 1];
+        return result;
+    },
+
+    /**
+     * Takes a string as used for storing an array of strings in the database, and transforms it back to an array of strings
+     * @param string
+     * @returns {array}
+     */
+    destringifyArrayOfStrings: function (string) {
```

Comment:
why not name this destringifyStringifiedArray? THis seems rather confusing to me